Fitness/Controllers/ClassListViewController.swift

```diff
@@ -155,18 +190,30 @@ class ClassListViewController: UIViewController {
         let filterNavController = UINavigationController(rootViewController: filterVC)
         tabBarController?.present(filterNavController, animated: true, completion: nil)
     }
-
-    func setupCollectionViews() {
+    
+    func initializeCollectionViews() {
         let calendarFlowLayout = UICollectionViewFlowLayout()
         calendarFlowLayout.itemSize = CGSize(width: 24, height: 47)
         calendarFlowLayout.scrollDirection = .horizontal
         calendarFlowLayout.minimumLineSpacing = 40.0
         calendarFlowLayout.sectionInset = .init(top: 0.0, left: 16.0, bottom: 0.0, right: 16.0)
 
         calendarCollectionView = UICollectionView(frame: .zero, collectionViewLayout: calendarFlowLayout)
-        calendarCollectionView.showsHorizontalScrollIndicator = false
         calendarCollectionView.delegate = self
         calendarCollectionView.dataSource = self
+        
+        let classFlowLayout = UICollectionViewFlowLayout()
+        classFlowLayout.itemSize = CGSize(width: UIScreen.main.bounds.width - 32.0, height: 100.0)
+        classFlowLayout.minimumLineSpacing = 12.0
+        classFlowLayout.headerReferenceSize = .init(width: UIScreen.main.bounds.width - 32.0, height: 72.0)
```

Comment:
can you use `view.bounds.width` instead?

Reply:
Since this is being called from the initializer using `view` causes `viewDidLoad` to be called, which crashes because the initializer didn't finish. Do you think I should structure this differently so that I can use `view.bounds.width`?

Reply:
No, it is ok for now. Just curious